kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -542,6 +559,19 @@ public void loadNextGenSearchPanel() {
         setupSlideOutTrayPane(nextGenSearchPanel, nexGenSearchSlideoutTrayPane);
     }
 
+    /**
+     * Add a Next Gen Reasoner Search Results, currently tied to the "bell" left nav button
+     */
+    public void loadNextGenReasonerPanel() {
+        Config nextGenReasonerConfig = new Config(NextGenReasonserController.class.getResource(NEXT_GEN_REASONER_FXML_URL));
+        JFXNode<Pane, NextGenReasonserController> reasonerJFXNode = FXMLMvvmLoader.make(nextGenReasonerConfig);
```

Comment:
(Optional) Use make() method when Config is small.

Could we specify `NextGenReasonserController.class.getResource(NEXT_GEN_REASONER_FXML_URL)` when calling the `make()` method?
```java
JFXNode<Pane, NextGenReasonserController> reasonerJFXNode = FXMLMvvmLoader.make(NextGenReasonserController.class.getResource(NEXT_GEN_REASONER_FXML_URL));
```

Reply:
fixed